controllers/external/tracker_test.go

```diff
@@ -64,7 +66,7 @@ func (c *watchCountController) Watch(_ source.Source) error {
 func TestRetryWatch(t *testing.T) {
 	g := NewWithT(t)
 	ctrl := newWatchCountController(true)
-	tracker := ObjectTracker{Controller: ctrl}
+	tracker := ObjectTracker{Controller: ctrl, Scheme: runtime.NewScheme(), Cache: &informertest.FakeInformers{}}
```

Comment:
Is there any better way to set cache for testing?

Reply:
I don't think so. The only alternatives I see is to either create a "real" cache or implement our own fake cache.

I would keep the current implementation